### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by mcousillas6**
## Just review the latest commit.
* Added docs on readme.
* Added license file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,88 @@
-# WIP
-## React native LINE
+# React native LINE
+iOS and Android Native wrapper for Line's MobileSDK.
+
+## Requirements
+- React native `0.48.+`.
+- LineSDK iOS `4.0.3` and Android `4.0.6`.
```
Comment (m-revetria) [inline README.md]: What about iOS requirements?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,88 @@
...
+
+## Installation
+First, install the npm package and link it to your Android and iOS projects with react-native link.
+```bash
+  yarn add react-native-line
```
Comment (m-revetria) [inline README.md]: maybe you can use npm as it is the default package manager for node. Thoughts?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,88 @@
...
+  react-native link react-native-line
+```
+### iOS Setup
+Follow all the configuration steps in [Line's iOS integration guide](https://developers.line.me/en/docs/line-login/ios/integrate-line-login/)
+
+### Android Setup
+1. Follow all the configuration steps in [Line's Android integration guide](https://developers.line.me/en/docs/line-login/android/integrate-line-login/)
+2. Add the string `line_channel_id` to your strings file with the the channel id that you have on your line console.
+```xml
+<string name="line_channel_id">Your channel id here</string>
```
Comment (m-revetria) [inline README.md]: please add translatable="false"

File: README.md (hunk 1 above)
Comment (mcousillas6) [reply README.md]: do you mean minimum iOS version?

File: README.md (hunk 2 above)
Comment (mcousillas6) [reply README.md]: Yep, sounds good

**Pull request merged**